frontend/javascripts/admin/user/permissions_and_teams_modal_view.js

```diff
@@ -0,0 +1,271 @@
+// @flow
+import { Modal, Button, Radio, Col, Row, Checkbox } from "antd";
```

Comment:
Git did not understand that I renamed the file and manipulated it :/

Reply:
Thanks for notifying me, I used http://www.mergely.com/editor to compare the two files which worked very well :)